src/scripts/generate-db.sql

```diff
@@ -0,0 +1,52 @@
+-- Database: osw
+
+-- DROP DATABASE IF EXISTS osw;
+
+CREATE DATABASE osw
+    WITH
+    OWNER = tdeiadmin
+    ENCODING = 'UTF8'
+    LC_COLLATE = 'en_US.utf8'
+    LC_CTYPE = 'en_US.utf8'
+    TABLESPACE = pg_default
+    CONNECTION LIMIT = -1
+    IS_TEMPLATE = False;
+
+-- Table: public.osw_versions
+
+-- DROP TABLE IF EXISTS public.osw_versions;
+
+CREATE TABLE IF NOT EXISTS public.osw_versions
+(
+    id integer NOT NULL DEFAULT nextval('osw_versions_id_seq'::regclass),
+    tdei_record_id character varying COLLATE pg_catalog."default" NOT NULL,
+    confidence_level integer DEFAULT 0,
+    tdei_org_id character varying COLLATE pg_catalog."default" NOT NULL,
+    tdei_service_id character varying COLLATE pg_catalog."default" NOT NULL,
```

Comment:
remove this. `OSW` does not have `tdei_service_id`

Reply:
fixed